### PR TITLE
Add menu bar buttons

### PR DESCRIPTION
**Pull request by GarrySan**

### CODE REVIEW

**Comment by GarrySan** [comment]
#3

File: src/Taskbar.tsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+
+class Taskbar extends React.Component {
+  clearMarkers() {
```
Comment (aloksingh3112) [inline src/Taskbar.tsx]: make this an arrow function as with onClick the "this" property changes this doesn't throws any error for now but we will definitely be using the this property in these functions.

File: src/Taskbar.tsx [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    console.log('clearing markers')
+  }
+  addRaw() {
```
Comment (aloksingh3112) [inline src/Taskbar.tsx]: same here make it a  arrow function.

File: src/Taskbar.tsx [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    console.log('Adding raw observations')
+  }
+  pull() {
```
Comment (aloksingh3112) [inline src/Taskbar.tsx]: make it a  arrow function.

File: src/Taskbar.tsx [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+    console.log('Pulling from google sheets')
+  }
+  refresh() {
```
Comment (aloksingh3112) [inline src/Taskbar.tsx]: make it a  arrow function.

File: src/Taskbar.tsx [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+    console.log('Refreshing markers')
+  }
+  openSheet() {
```
Comment (aloksingh3112) [inline src/Taskbar.tsx]: make it a  arrow function.

**Comment by aloksingh3112** [comment]
Also @ivanoats as we are on the recent version of react we should be using the best strategies that current version of react has, like we can use react-hooks to decouple our logic from our component we can use the power of custom hooks for that.
Also with react hooks code looks more cleaner and all the trouble of handling state in most of the component life cycle hooks is removed.

Also about the css if we are planning to use react-material-ui in project we wont be requiring any external css files that fixes alignment and all in hacky ways.
Also react-material-ui is very much customizable so its a win win situation for us if we use that.

**Comment by ivanoats** [comment]
I agree - function components and hooks if possible.

**Comment by ivanoats** [comment]
This PR needs to be closed, or refactored from a class based component to a functional / hooks style.

**Comment by sahil-shubham** [comment]
Hi @ivanoats I would like to work on this issue. This PR does not incorporate react hooks, and it would be better if I open another PR for the issue #3 rather than trying to solve the merge conflicts.

**Pull request closed without merging**